src/graph.py

```diff
@@ -635,6 +640,56 @@ def grid_search(self, prize_file, Gs, Bs, Ws):
         return forest, augmented_forest, params_by_nodes
 
 
+    def _eval_randomizations(self, params):
+        """
+        Convenience methods which sets parameters and performs PCSF
+        """
+
+        self._reset_hyperparameters(params)
+        paramstring = "G_{}_B_{}_W_{}".format(*[int(x) if int(x) == x else x for x in [params['g'], params['b'], params['w']]])
+        logger.info("Randomizations for " + paramstring)
+        logger.info(params)
+
+        forest, augmented_forest = self.randomizations(params["noisy_edges_repetitions"], params["random_terminals_repetitions"])
+        # prune_network_graph(forest); prune_network_graph(augmented_forest)
+        
+        return paramstring, forest, augmented_forest
+
+
+    def _grid_randomization(self, prize_file, params):
+        """
+        Internal function which executes pcsf at every point in a parameter grid.
+        Subroutine of `grid_search`.
+
+        Arguments:
+            prize_file (str): filepath
+            Ws (list): Values of omega
+            Bs (list): Values of beta
+            Gs (list): Values of gamma
```

Comment:
The arguments list here does not correspond with the arguments list the function takes.